.tav.yml

```diff
@@ -221,7 +221,7 @@ apollo-server-express-2_12:
   # but only the latest MAJOR.MINOR.x to reduce the test matrix.
```

Comment:
nit: can we update the "We want this version range: ..." comment to say why we are clipping at >2.8?

Reply:
meh, nevermind. Encoded in the PR discussion here.

Reply:
Update the versions in the comments, thanks for catching that.